### PR TITLE
Login

### PR DESCRIPTION
**Pull request by farouqdaif1**
## What does this PR do?
This PR is raised to Build a web app that requires the user to log in.
 
## Description of the task to be completed
* [x]  [Install and setup devise](https://github.com/heartcombo/devise#getting-started).

   * [x]  Can register a new user.
   * [x]  User logs in with a combination of email and password.
   * [x]  Hashed password should be stored in the database.
   * [x]  Ask for confirmation of email.
   * [x]  Can reset password.
 
 ## How should it be tested
 Clone the repository
 
 ```
 git clone 
 ```
 Navigate to the folder
 
 ```
 cd Blog2
 ```
 Checkout the branch
 
 ```
 git checkout Login
 ```
 
 #### Thanks for reviewing my PR 😊

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (arslanbisharat) [inline app/models/user.rb]: - [x] Please make sure `:confirmable` is enabled in the user model

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @farouqdaif1,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 1 above)
Comment (farouqdaif1) [reply app/models/user.rb]: Thanks I will do it

**Review by Lordkaito — APPROVED** [review]
Hi @farouqdaif1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**